Gruntfile.js

```diff
@@ -39,8 +39,8 @@ var sass = require('sass');
         // Function to get all scripts for use with a given theme
         var scriptPaths = function(theme) {
             var files = [
-                'node_modules/jquery/dist/jquery.min.js',
-                'node_modules/chosen-npm/public/chosen.jquery.min.js',
+                'node_modules/jquery/dist/jquery.js',
+                'node_modules/chosen-npm/public/chosen.jquery.js',
```

Comment:
We probably should use the minimised version of jQuery.

Reply:
Agreed, switched to this to make readings some console errors easier. Reverting to min.